PhpUnit/FunctionalTestBootstrap.php

```diff
@@ -1,30 +1,28 @@
 <?php
-namespace Neos\Flow\Build;
 
 /*
- * This file is part of the Neos Flow build system.
- *
  * (c) Contributors of the Neos Project - www.neos.io
- *
- * This package is Open Source Software. For the full copyright and license
- * information, please view the LICENSE file which was distributed with this
- * source code.
+ * Please see the LICENSE file which was distributed with this source code.
  */
 
-$context = isset($_SERVER['FLOW_CONTEXT']) ? $_SERVER['FLOW_CONTEXT'] : 'Testing';
+declare(strict_types=1);
+
+namespace Neos\Flow\Build;
+
+use Neos\BuildEssentials\TestableFramework;
+use Neos\Flow\Core\Bootstrap;
+use Neos\Flow\Testing\RequestHandler\RuntimeSequenceHttpRequestHandler;
+
+$context = TestableFramework::getApplicationContext();
 
 if (preg_match('/^(?:Testing|Testing\/.+)$/', $context) !== 1) {
 	die(sprintf('The context "%s" is not allowed. Only "Testing" context or one of its subcontexts "Testing/*" is allowed.', $context));
 }
 
-$_SERVER['FLOW_ROOTPATH'] = dirname(__FILE__) . '/../../../';
-
-if (DIRECTORY_SEPARATOR === '/') {
-	// Fixes an issue with the autoloader, see FLOW-183
-	shell_exec('cd ' . escapeshellarg($_SERVER['FLOW_ROOTPATH']) . ' && FLOW_CONTEXT=' . escapeshellarg($context) . ' ./flow neos.flow:cache:warmup');
-}
```

Comment:
thanks for removing that lol. i found this line as well some time and was just whuuuut?

Reply:
I tried it without, whatever that issue with the autoloader was, I am going to say that was before composer and doesn't matter anymore. It works just fine without.